### PR TITLE
Added note on oidc re auth0

### PR DESCRIPTION
**Pull request by stevenGravy**
Note on turning off OIDC Conformant settings that prevents auth0 connectors from populating claims.  Otherwise a warning will show like "Error validating callback: unable to map claims to role for connector:" in the log.

### CODE REVIEW

File: docs/7.x/config.md [hunk 1]
```diff
@@ -404,6 +404,8 @@ spec:
     - {% raw %}{claim: "roles", value: "gravitational/admins", roles: ["@teleadmin"]}{% endraw %}
 ```
 
+!!! note For Auth0 the OIDC Conformant setting should be off in Advanced Setting -> OAuth or Claims will not populate properly
```
Comment (r0mant) [inline docs/7.x/config.md:407]: Nit: Could you add a dot at the end of the sentence please?

Also, shouldn't the note be formatted like this:

```
!!! note:
  <note text>
```

Disregard if your way is displayed properly as well.

**Review by r0mant — APPROVED** [review]

File: docs/7.x/config.md (hunk 1 above)
Comment (stevenGravy) [reply docs/7.x/config.md]: think it would still work but modified to make it uniform with other notes in the doc

File: docs/7.x/config.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Resource Name             | Resource Description
 `role`                    | User role (Enterprise only)
 `user`                    | Cluster user
 `token`                   | User tokens such as API keys
-`logforwarder`            | Forwarding logs to a remote rsyslog server
+`logforwarder`            | Forwarding logs to a remote rsyslog server!
```
Comment (a-palchikov) [inline docs/7.x/config.md:16]: Why an exclamation mark here?

Comment (stevenGravy) [reply docs/7.x/config.md]: that was an error, thanks for catching @a-palchikov !  it has been removed

**Review by a-palchikov — APPROVED** [review]

**Pull request merged**